ARCore/app/src/main/java/com/novoda/spikes/arcore/poly/PolyApi.kt

```diff
@@ -0,0 +1,100 @@
+package com.novoda.spikes.arcore.poly
+
+import android.net.Uri
+import android.util.Log
+import com.squareup.moshi.KotlinJsonAdapterFactory
+import com.squareup.moshi.Moshi
+import okhttp3.*
+import java.io.IOException
+
+class PolyApi {
+
+    companion object {
+        private const val HOST = "poly.googleapis.com"
+        private const val VERSION = "v1"
+        private const val API_KEY = "API_KEY_HERE"
```

Comment:
seems like a nice use case for https://github.com/novoda/gradle-build-properties-plugin 😄

Reply:
Might be overkill here, it's just a sample